ciudadano_app/models/ciudadano/ciudadano.py

```diff
@@ -78,7 +81,7 @@ def obtener_identificacion(self):
         """Retorna el número de identificación del ciudadano"""
         return self.numero_identificacion
 
-    def __str__(self):
+    def _str_(self):
```

Comment:
El metodo es __str__ c:
```suggestion
    def __str__(self):
```

Reply:
listo